src/core/sys/linux/sys/time.d

```diff
@@ -0,0 +1,72 @@
+/**
+ * D header file for POSIX.
+ *
+ * Copyright: Copyright Sociomantic Labs GmbH.
+ * License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Authors:   Leandro Lucarella
+ */
+
+/*          Copyright Sociomantic Labs GmbH.
+ * Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
+ */
+module core.sys.linux.sys.time;
+
+private import core.sys.linux.config;
+public import core.sys.posix.sys.types;  // timeval
+
+version (linux)
+else static assert(false, "Unsupported platform");
+
+version( CRuntime_Glibc )
+else static assert(false, "Unsupported platform");
```

Comment:
There's a trivial error here, you need empty braces for the `version` statement to apply to something.

But I'm wondering if we need all this anyway. Looking at other examples in the sys/linux directory, I think you can just put:
```D
version (linux): version (CRuntime_Glibc):
```
Having imports that explode a compilation means it just pushes the version statements outside the lib and into user code.